modules/sdk-coin-avaxp/src/lib/delegatorTxBuilder.ts

```diff
@@ -124,6 +124,10 @@ export class DelegatorTxBuilder extends TransactionBuilder {
    *   unix time stamp based off seconds
    */
   validateStakeDuration(startTime: BN, endTime: BN): void {
+    const oneDayLater = new BN(Date.now()).add(new BN(86400));
```

Comment:
Wouldn't the validation need to be against an implemented DelegatorTxBuilder's startTime and not one built as a one off?

Reply:
```suggestion
    const oneDayLater = new BN(Date.now()).sub(new BN(86400));
```